packages/token-tools/src/types.ts

```diff
@@ -466,6 +466,7 @@ export interface ShadowValueNormalized {
   offsetY: DimensionValue;
   blur: DimensionValue;
   spread: DimensionValue;
+  inset?: boolean;
```

Comment:
Let’s just enforce this in the normalized version. The intent with the normalized versions is to resolve all values, fill in defaults, and give a final, predictable shape. We can be much stricter here than the spec, because we (the parser) are the one that makes this! So we’re only helping ourselves (and plugins) out by normalizing what we can.

```suggestion
  inset: boolean;
```